app/src/components/job/hr/UpdateJob.jsx

```diff
@@ -49,78 +51,83 @@ export default function HRUpdateJob() {
     setJobIsActive(!jobIsActive);
   };
 
-  return (
-    <div className='relative flex flex-col container max-w-7xl mt-10 bg-white p-10 mx-auto rounded-lg shadow-lg shadow-blue-200'>
-      <h1 className='text-3xl text-left font-bold'>Update Job</h1>
-      <form onSubmit={handleSubmit} className='pt-10'>
-        <div className='mb-6'>
-          <label
-            htmlFor='jobName'
-            className='block mb-2 text-md font-medium text-gray-900 dark:text-gray-300 space-y-2'
-          >
-            <p>Job Name</p>
-            <p className='italic font-light text-gray-400 text-sm'>
-              At least 1 character, maximum 50 characters. Whitespaces at the front and back will be
-              removed.
-            </p>
-            <input
-              type='text'
-              id='jobName'
-              value={jobName}
-              onChange={handleJobNameChange}
-              className='bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5'
-              placeholder='e.g. Sales Engineer'
-            />
-          </label>
+  switch (currentUserType) {
+    case "HR":
+      return (
+        <div className='relative flex flex-col container max-w-7xl mt-10 bg-white p-10 mx-auto rounded-lg shadow-lg shadow-blue-200'>
```

Comment:
It would be good if we could have a way to break down this component. Seems a bit big.